src/styles/variables/variables-z-index.less

```diff
@@ -35,6 +35,8 @@ Z-Index Depths
 @z-index-side-panel-container:                                                  @z-index-side-panel + 4;
 @z-index-side-panel-expand-button:                                              @z-index-side-panel + 10;
 @z-index-sidebar:                                                               @z-index-header;
+// The error message comes in 'body' and 'modal' varieties
+@z-index-errormsg-floater-modal:												@z-index-modal-container + 1;
```

Comment:
Let's add more spaces to get the value of the variable to be at the 80th column.